qiskit_addon_dice_solver/dice_solver.py

```diff
@@ -231,7 +236,14 @@ def solve_fermion(
         - Approximate ground state from SCI
         - Average orbital occupancy
     """
-    ci_strs = bitstring_matrix_to_ci_strs(bitstring_matrix, open_shell=open_shell)
+    if isinstance(bitstring_matrix, Sequence):
```

Comment:
```suggestion
    if isinstance(bitstring_matrix, tuple):
```
I prefer this, and it's what we do at qiskit-addon-sqd.

Reply:
I had considered letting Sequence[Sequence[int]] slide through, even though that's not what the type hint says. I don't mind being more strict if you think it's best